docs/api.md

```diff
@@ -0,0 +1,72 @@
+# Whitehall API
+
+Whitehall is primarily a publishing application but it also provides an API used
+by some applications. This API is public and is served under
+[https://www.gov.uk/api/](https://www.gov.uk/api/governments).
+
+Whitehall is not the only application to provide APIs under this path. For
+example, [/api/content](https://www.gov.uk/api/content/government) is served by
+Content Store and [/api/search.json](https://www.gov.uk/api/search.json) is
```

Comment:
You might wish to link to https://content-api.publishing.service.gov.uk for the Content API docs.